web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1720,3 +1720,32 @@ def handle_import(args):
         sys.exit(1)
 
     LOG.info("Import successful!")
+
+
+def handle_labels(args):
+    if 'list' in args:
+        checker_labels = CheckerLabels(
+            os.path.join(os.environ['CC_PACKAGE_ROOT'], 'config', 'labels'))
```

Comment:
I am getting an exception when I run the following command: `CodeChecker cmd labels --list`
```
Traceback (most recent call last):
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_common/cli.py", line 175, in main
    sys.exit(args.func(args))
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_client/cmd_line_client.py", line 1728, in handle_labels
    os.path.join(os.environ['CC_PACKAGE_ROOT'], 'config', 'labels'))
  File "/usr/lib/python3.6/os.py", line 669, in __getitem__
    raise KeyError(key) from None
KeyError: 'CC_PACKAGE_ROOT
```

```suggestion
            os.path.join(os.environ['CC_DATA_FILES_DIR'], 'config', 'labels'))
```

After fixing it I will get another exception:
```
Traceback (most recent call last):
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_common/cli.py", line 175, in main
    sys.exit(args.func(args))
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_client/cmd_line_client.py", line 1746, in handle_labels
    rows = [[label] for label in checker_labels.labels()]
AttributeError: 'CheckerLabels' object has no attribute 'labels'
```

It would be good to add a cmdline test case which calls this command and test whether it returns with a correct output or not.